src/vs/editor/contrib/suggest/browser/media/suggest.css

```diff
@@ -23,7 +23,13 @@
 
 .monaco-editor .suggest-widget > .tree {
 	height: 100%;
-	width: 100%;
+	width: 40%;
+	float: left;
```

Comment:
Careful with floats. If all you have is two divs, you should use the usual `inline-block` display to lay them out side by side.